### PR TITLE
343846 migration update

### PR DESCRIPTION
**Pull request by AtosSK**

### CODE REVIEW

File: src/EPR.Accreditation.Common.Data/Migrations/20240510113820_Country-Code.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+namespace EPR.Accreditation.API.Common.Data.Migrations
+{
+    public partial class CountryCode : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "CountryCode",
+                table: "Accreditation",
+                type: "nvarchar(max)",
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Common.Data/Migrations/20240510113820_Country-Code.cs:14]: Shouldn't be max. How many characters is country code going to be?

File: src/EPR.Accreditation.Common.Data/Migrations/AccreditationContextModelSnapshot.cs [hunk 2]
```diff
@@ -36,6 +36,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("AccreditationStatusId")
                         .HasColumnType("int");
 
+                    b.Property<string>("CountryCode")
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Common.Data/Migrations/AccreditationContextModelSnapshot.cs:39]: As above

File: src/EPR.Accreditation.Common.Data/Migrations/20240510113820_Country-Code.cs (hunk 1 above)
Comment (AtosSK) [reply src/EPR.Accreditation.Common.Data/Migrations/20240510113820_Country-Code.cs]: Done.

File: src/EPR.Accreditation.Common.Data/Migrations/AccreditationContextModelSnapshot.cs (hunk 2 above)
Comment (AtosSK) [reply src/EPR.Accreditation.Common.Data/Migrations/AccreditationContextModelSnapshot.cs:39]: Done.

**Pull request closed without merging**